### PR TITLE
[ROCKETMQ-44]  Refactor to avoid duplicated codes in DefaultMessageStore

### PR DESCRIPTION
**Pull request by wu-sheng**
Refactor methods and create a new method. And format code by the given code-style.

### CODE REVIEW

**Comment by wu-sheng** [comment]
ref issue https://issues.apache.org/jira/browse/ROCKETMQ-44

**Comment by zhouxinyu** [comment]
Hi @wu-sheng, thanks for you PR, could please use the rocketmq code-style to reformat these commits?

As for
```java
} catch (Exception e) {
}
```

or

```java
}
catch (Exception e) {
}
```
, RocketMQ choose the first style, please refer to [here](http://rocketmq.incubator.apache.org/docs/code-guidelines/). The same issue related to `else`, `else if`...

**Comment by wu-sheng** [comment]
@zhouxinyu, seems your style cause this. My intelliJ config is like this:

wx20170113-165204 2x

**Comment by wu-sheng** [comment]
@zhouxinyu , All travis-ci passed. Why style is still not correct?

**Comment by zhouxinyu** [comment]
@wu-sheng , sorry for my fault, I made some changes in intelliJ graph config page which doesn't reflect to the style file, anyway, I have updated the code style file, please check it.

**Comment by wu-sheng** [comment]
Ok, I will sync this change to my pull requests. And check later.

**Comment by wu-sheng** [comment]
Sorry, seem still not work. The close-brace style is OK, but `typecast`-style is still incorrect.
As I already mention on https://github.com/apache/incubator-rocketmq/pull/36, please check on that.

This is my local log:
```shell
[INFO] --- maven-checkstyle-plugin:2.17:check (verify) @ rocketmq-namesrv ---
[INFO] Starting audit...
/Users/wusheng/Documents/code/github/incubator-rocketmq-fork/namesrv/src/main/java/org/apache/rocketmq/namesrv/NamesrvStartup.java:103:47: error: 'typecast' is not followed by whitespace.
Audit done.

```

**Comment by wu-sheng** [comment]
@zhouxinyu , just did it manually. It's crazy....

**Comment by zhouxinyu** [comment]
Hi @wu-sheng ,Could you please add some unit tests for this PR ?

**Comment by wu-sheng** [comment]
@zhouxinyu , I have tried before. But there is a problem. In recent RocketMQ codes, there are no `ConsumeQueueMock` and `CommitLogMock`, I am not sure if I create the Mock-Classes, they are good enough.

If you want me to do that, I can try it, but no guarantee. Any thoughts? @vongosling @WillemJiang

**Comment by wu-sheng** [comment]
@zhouxinyu , If your team member can provide these Mock-Classes, it will be easier to add test-cases for my changes.

**Comment by vongosling** [comment]
@wu-sheng Thanks for your clean code actions. Why so many format operation, what had happend when you use our checkstyle, let me know. BTW, you know, UT is the best assistant of the refactoring :-)

File: store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java [hunk 1]
```diff
@@ -1278,7 +1282,8 @@ public void doDispatch(DispatchRequest req) {
         }
     }
 
-    public void putMessagePositionInfo(String topic, int queueId, long offset, int size, long tagsCode, long storeTimestamp,
+    public void putMessagePositionInfo(String topic, int queueId, long offset, int size, long tagsCode,
+        long storeTimestamp,
```
Comment (wu-sheng) [inline store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java]: @vongosling , I am not sure what in checkstyle trigger this. And what is **UT**? Sorry, no clue. 😝

**Pull request closed without merging**